### PR TITLE
Update docs to match template refactor

### PR DESCRIPTION
**Pull request by marrobi**
# PR for issue #942
## What is being addressed

Update the docs to match refactored bundles.

## How is this addressed

- Remove redundant docs, and remove redundant innereye bundles, update existing docs
- rename workspace-templates to templates as includes more than workspace templates
- fix a couple of env.sample files

##  Follow on work:
#962

### CODE REVIEW

File: docs/templates/workspace-services/inner-eye.md [hunk 1]
```diff
@@ -71,3 +72,33 @@ URLs:
 
 1. The runner will provide you with a link and ask you to open it to login. Copy the link and open it in browser (Edge) on the DSVM and login. The run will continue after login.
 1. In your browser navigate to ml.azure.com and open the `Experiments` tab to follow the progress of the training
+
+## Configuring and testing inference service
+
+The workspace service provision an App Service Plan and an App Service for hosting the inference webapp. The webapp will be integrated into the workspace network, allowing the webapp to connect to the AML workspace. Following the setup you will need to:
```
Comment (tompaana) [inline docs/templates/workspace-services/inner-eye.md:78]:
```suggestion
The workspace service provisions an App Service Plan and an App Service for hosting the inference webapp. The webapp will be integrated into the workspace network, allowing the webapp to connect to the AML workspace. Following the setup you will need to:
```

File: docs/templates/workspaces/base.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Azure TRE base workspace
+
+The base workspace template is the foundation that all other workspaces and workpace services are built upon. Alternative workspace architectures could be used however the tempaltes provided in this repository rely on the specific architecture of this base workspace.
+
+The base workspace template contains the following resources:
+
+- Virtual Network
+- Storage Account
+- KeyVault
```
Comment (tompaana) [inline docs/templates/workspaces/base.md:9]:
```suggestion
- Key Vault
```

File: docs/templates/workspaces/inner-eye.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  ```cmd
+  make porter-build DIR=./templates/workspace_services/azureml
+  make porter-publish DIR=./templates/workspace_services/azureml
+  ```
+
+1. Create a copy of `workspaces/innereye/.env.sample` with the name `.env` and update the variables with the appropriate values.
+
+  | Environment variable name | Description |
+  | ------------------------- | ----------- |
```
Comment (tompaana) [inline docs/templates/workspaces/inner-eye.md:31]:
```suggestion
```

File: docs/tre-admins/registering-templates.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
-# Registering Workspace Templates
+# Registering Templates
 
-To deploy a new type of Workspace, we need to register a Workspace Template using the API.
+To enable users to deploy Workspaces, Workspace Services or User Resources, we need to register their Templates using the API.
 
 ## Porter Bundles
 
-Porter bundles can either be registered interactively using the Swagger UI or automatically using the utility script (useful in CI/CD scenarios). The script is provided at `/devops/scripts/publish_register_bundle.sh`.
+Templates are envapsulated in [Porter](https://porter.sh) bundles. Porter bundles can either be registered interactively using the Swagger UI or automatically using the utility script (useful in CI/CD scenarios). The script is provided at `/devops/scripts/publish_register_bundle.sh`.
```
Comment (tompaana) [inline docs/tre-admins/registering-templates.md:7]:
```suggestion
Templates are encapsulated in [Porter](https://porter.sh) bundles. Porter bundles can either be registered interactively using the Swagger UI or automatically using the utility script (useful in CI/CD scenarios). The script is provided at `/devops/scripts/publish_register_bundle.sh`.
```

File: docs/tre-admins/registering-templates.md [hunk 5]
```diff
@@ -13,11 +13,12 @@ The script can also be used to generate the payload required by the API without
 
 ### Registration using Swagger UI
 
-1. We will use the utility script to generate the payload. The script needs to be executed from within the bundle directory, for example `/templates/workspaces/azureml_devtestlabs/`. This script can be used as follows:
+1. We will use the utility script to generate the payload. The script needs to be executed from within the bundle directory, for example `/templates/workspaces/base/`. This script can be used as follows:
 
    ```cmd
-   ../../../devops/scripts/publish_register_bundle.sh -r <acr_name> -i -t workspace
+   ../../../devops/scripts/publish_register_bundle.sh -r <acr_name> -i -t workspace>
```
Comment (tompaana) [inline docs/tre-admins/registering-templates.md:19]: Why the angle bracket at the end?

**Review by tompaana — APPROVED** [review]
Few suggestions to fix typos etc. Other than that, looks very good!

File: docs/tre-admins/registering-templates.md (hunk 5 above)
Comment (marrobi) [reply docs/tre-admins/registering-templates.md]: Fixed

**Pull request merged**